README.md

```diff
@@ -2,24 +2,220 @@
+Participation in the consortium is both formal and informal:
 
-## Building the PMIx Standard document
+* Informal participation is encouraged by working with the PMIx Standard community and/or one of the PMIx implementation libraries in an _ad hoc_ manner, such as (but not limited to) engaging in discussions on the mailing lists, reporting testing results, providing feature requests and bug reports, filing pull requests for proposed modifications, etc. Informal participants do not have voting privileges on administrative matters, but are encouraged to attend the quarterly meetings and participate in those discussions.
 
-You will need a LaTeX installation on your system.
-To build simply type `make` and it will generate the standard PDF.
-See documentation in header comment of `pmix-standard.tex`, and the macros provided by `pmix.sty`.
+* Organizations become eligible for elevation to Membership upon having at least one representative attend two quarterly meetings within the prior twelve months. All new Members must either request membership or be nominated by an existing ASC Member, and be approved by the ASC according to the voting rules described below. Membership confers representation on the ASC, thus granting the right to participate in formal votes, along with public acknowledgment for the entity as being a Member of the PMIx Standard ASC on the PMIx web site. Each Member organization is allowed one "seat" on the ASC, to be filled as the Member chooses. The official representative for an organization is responsible for notifying the ASC in advance of an alternate should the representative not be attending a meeting in order to confer voting rights to that individual.
+    * Note: Individuals may also become Members so long as their direct employer is not a current Member of the ASC. For these purposes, a consultant is not considered a direct employee - thus, someone consulting for a current ASC Member is eligible to become an individual member should they choose to do so. Individual members who subsequently join Member organizations will be absorbed into the larger organization's membership. This likewise applies to individuals who are affiliated with an existing Member organization (e.g., as a contractor), but are not direct employees of that organization. Individuals requesting membership that are affiliated with multiple ASC members can be granted individual Member status by vote of the ASC. This is intended to preclude undue influence in voting procedures by an organization with multiple individual members.
+
+* A Member is removed from the ASC (a) upon having failed to have at least one representative attend two quarterly meetings in the prior twelve months, or (b) by vote of the other ASC members per the voting rules defined below. Attendance by a non-voting member of the organization (i.e., someone not officially designated as an alternate for voting purposes) counts towards meeting this requirement.
```

Comment:
>  Does that mean that 2/3 of members may kick another member if they want to?

Yes! Standards organizations typically protect themselves with a clause such as this - e.g., for the case where someone or some organization is deliberately using the standard as a competitive "weapon" or otherwise behaving in an unprofessional/disruptive manner.